### PR TITLE
390 Add an API to return session chairs and runners

### PR DESCRIPTION
**Pull request by dpoirier**
Requires an API key; see the API docs once they've
been updated.

### CODE REVIEW

File: pycon/schedule/views.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import hashlib
+import json
```
Comment (dchukhin) [inline pycon/schedule/views.py]: is this import used in this file?

File: pycon/schedule/views.py [hunk 2]
```diff
@@ -7,6 +7,9 @@
 from django.contrib.auth.models import User
 
 from .models import Session, SessionRole
+from pycon.pycon_api.decorators import api_view
+from symposion.schedule.models import Slot
+from symposion.schedule.views import json_serializer
```
Comment (dchukhin) [inline pycon/schedule/views.py]: is this import used?

**Comment by dpoirier** [comment]
Thanks @dchukhin I've removed those unused imports, and I ran flake8 on the files that were modified in this pull and cleaned up the other things it spotted.

**Comment by dchukhin** [comment]
looks good :ok:

**Pull request merged**